bullet-train.zsh-theme

```diff
@@ -591,9 +594,17 @@ prompt_status() {
   fi
 
   local symbols
+  local ret_signal
   symbols=()
+  ret_signal=$(kill -l $(($RETVAL - 128)))
```

Comment:
Can this be inside 
`if [[ $RETVAL -gt 128 && $BULLETTRAIN_STATUS_EXIT_SIGNAL_SHOW == true ]]; the`
?